larastan.neon

```diff
@@ -4,12 +4,15 @@ includes:
     - ../../larastan/larastan/extension.neon
 
 parameters:
-    checkMissingIterableValueType: false
     namespaceAndSuffix:
         App\Events: Event
         App\Listener: Listener
         App\Policies: Policy
         App\Jobs: Job
+    ignoreErrors:
+        -
+            identifier: missingType.iterableValue
+            reportUnmatched: false
```

Comment:
I think it's probably safe to either ignore errors with this identifier (and don't report unmatched), or to not ignore the error at all (which is my preference).